build/docker/Release.dockerfile

```diff
@@ -1,17 +1,31 @@
+# This Dockerfile creates a production release image for the project. This
+# downloads the release from releases.hashicorp.com and therefore requires that
+# the release is published before building the Docker image.
+#
+# We don't rebuild the software because we want the exact checksums and
+# binary signatures to match the software and our builds aren't fully
+# reproducible currently.
 FROM alpine:latest
 
-ARG NAME=vault-k8s
-ARG VERSION=0.3.0
+# NAME and VERSION are the name of the software in releases.hashicorp.com
+# and the version to download. Example: NAME=consul VERSION=1.2.3.
+ARG NAME
+ARG VERSION
+ARG LOCATION
+
+LABEL maintainer="Vault Team <vault@hashicorp.com>"
+LABEL version=$VERSION
 
 # Set ARGs as ENV so that they can be used in ENTRYPOINT/CMD
 ENV NAME=$NAME
 ENV VERSION=$VERSION
 
 # This is the location of the releases.
-ENV HASHICORP_RELEASES=https://releases.hashicorp.com
+ENV LOCATION=$LOCATION
 
-RUN addgroup vault && \
-    adduser -S -G vault vault
+# Create a non-root user to run the software.
+RUN addgroup ${NAME} && \
+    adduser -S -G ${NAME} ${NAME}
```

Comment:
This should be `vault`, not `vault-k8s`.

Reply:
Thanks, updated.